[CI-fails]: ModuleNotFoundError: No module named 'ipython_genutils'

Currently we expreince a issue with the automated testing. After the deprecation waring at the beginning (this should already be fixed by https://github.com/OpenEnergyPlatform/oeplatform/pull/885/commits/e666e3bb2f02d7dc33f2387c48ee9038a1038883) the report suggests that there is the module `python_genutils` missing. This module is called by the module `nbconvert` wich is part of the tutorials django-app. I will try to manually install the pip package `python_genutils` and/or update `nbconvert `to the latest version.


Bugreports: https://github.com/OpenEnergyPlatform/oeplatform/actions 


``` bash
Run cp oeplatform/securitysettings.py.default oeplatform/securitysettings.py
/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/rdflib_jsonld/__init__.py:12: DeprecationWarning: The rdflib-jsonld package has been integrated into rdflib as of rdflib==6.0.1.  Please remove rdflib-jsonld from your project's dependencies.
  DeprecationWarning,
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/django/core/management/__init__.py", line 4[19](https://github.com/OpenEnergyPlatform/oeplatform/runs/5752933823?check_suite_focus=true#step:8:19), in execute_from_command_line
    utility.execute()
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/django/core/management/__init__.py", line 413, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/django/core/management/base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/django/core/management/base.py", line 393, in execute
    self.check()
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/django/core/management/base.py", line 423, in check
    databases=databases,
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/django/core/checks/registry.py", line 76, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/django/core/checks/urls.py", line 13, in check_url_config
    return check_resolver(resolver)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/django/core/checks/urls.py", line 23, in check_resolver
    return check_method()
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/django/urls/resolvers.py", line 416, in check
    for pattern in self.url_patterns:
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/django/utils/functional.py", line 48, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/django/urls/resolvers.py", line 602, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/django/utils/functional.py", line 48, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/django/urls/resolvers.py", line 595, in urlconf_module
    return import_module(self.urlconf_name)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line [21](https://github.com/OpenEnergyPlatform/oeplatform/runs/5752933823?check_suite_focus=true#step:8:21)9, in _call_with_frames_removed
  File "/home/runner/work/oeplatform/oeplatform/oeplatform/urls.py", line 34, in <module>
    url(r"^tutorials/", include("tutorials.urls")),
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/django/urls/conf.py", line 34, in include
    urlconf_module = import_module(urlconf_module)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/importlib/__init__.py", line 1[27](https://github.com/OpenEnergyPlatform/oeplatform/runs/5752933823?check_suite_focus=true#step:8:27), in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 7[28](https://github.com/OpenEnergyPlatform/oeplatform/runs/5752933823?check_suite_focus=true#step:8:28), in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/runner/work/oeplatform/oeplatform/tutorials/urls.py", line 3, in <module>
    from tutorials import views
  File "/home/runner/work/oeplatform/oeplatform/tutorials/views.py", line 13, in <module>
    import nbconvert
  File "/opt/hostedtoolcache/Python/3.7.12/x[64](https://github.com/OpenEnergyPlatform/oeplatform/runs/5752933823?check_suite_focus=true#step:8:64)/lib/python3.7/site-packages/nbconvert/__init__.py", line 4, in <module>
    from .exporters import *
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbconvert/exporters/__init__.py", line 1, in <module>
    from .base import (export, get_exporter, 
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbconvert/exporters/base.py", line 14, in <module>
    from ipython_genutils.py3compat import string_types
ModuleNotFoundError: No module named 'ipython_genutils'
Error: Process completed with exit code 1.
```

--- a/.github/workflows/automated-testing.yaml
+++ b/.github/workflows/automated-testing.yaml
@@ -46,6 +46,7 @@ jobs:
     strategy:
       matrix:
         python-version:
+          -3.8
           -3.9
     steps:
       - uses: actions/checkout@v2

--- a/.github/workflows/automated-testing.yaml
+++ b/.github/workflows/automated-testing.yaml
@@ -46,8 +46,7 @@ jobs:
     strategy:
       matrix:
         python-version:
-          - 3.6
-          - 3.7
+          -3.9
     steps:
       - uses: actions/checkout@v2
         with:

--- a/.github/workflows/automated-testing.yaml
+++ b/.github/workflows/automated-testing.yaml
@@ -45,9 +45,7 @@ jobs:
     # We run this for python version 3.7.
     strategy:
       matrix:
-        python-version:
-          -3.8
-          -3.9
+        python-version: ['3.8', '3.9']
     steps:
       - uses: actions/checkout@v2
         with:

--- a/README.md
+++ b/README.md
@@ -12,7 +12,7 @@ This repository is licensed under [GNU Affero General Public License v3.0 (AGPL-
 
 ## Installation
 
-The installation steps have been proofed on linux and windows for python 3.6 and 3.7. Be aware that some of the required packages present installation's difficulties on windows
+The installation steps have been proofed on linux and windows for python 3.8 and 3.9. Be aware that some of the required packages present installation's difficulties on windows
 
 
 ### Setup the repository

--- a/requirements.txt
+++ b/requirements.txt
@@ -27,4 +27,4 @@ rdflib-jsonld # DeprecationWarning when rdflib>=6.0
 django_better_admin_arrayfield
 sparqlwrapper
 Pygments
-nbconvert==5.6.1
+nbconvert>=6.4.5

--- a/versions/changelogs/current.md
+++ b/versions/changelogs/current.md
@@ -1,9 +1,11 @@
 ### Changes
 - Update links on landing page (PR #896)
 - Update navigation bar (PR #898)
+- Python 3.6 is no longer supported due to dependency requirements (PR#941)
 
 ### Bugs
 - Prohibit duplicate / similar tags (PR #885)
+- Fix CI error by raising nbconvert package version, this required to drop support for Python 3.6 (PR#941)
 
 ### Features
 - Synchronize metadata keywords with tags (PR #885)

